internal/services/payments/payments.go

```diff
@@ -1,8 +1,108 @@
 package payments
 
-func AddAmount(addReq AddRequest, id string) (addRes AddResponse, err error) {
-	return addRes, err
+import (
+	"Tradeasy/config"
+	"Tradeasy/internal/model"
+	"bytes"
+	"encoding/json"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"time"
+)
+
+func AddAmount(addReq AddRequest, Userid string) (addRes AddResponse, err error) {
+	addAmount := addReq.Amount
+	var (
+		tradingAcc  model.TradingAccount
+		addResponse AddResponse
+		razorpayRes RazorpayResponse
+	)
+	if err = config.DB.Table("trading_account").Where("user_id = ?", Userid).First(&tradingAcc).Error; err != nil {
+		return addRes, err
+	}
+	username := "rzp_test_Oqf3eW39O728uq"
```

Comment:
initialise them as constants as of now in the same file. Will move to configuration later.